Dockerfile

```diff
@@ -24,6 +24,10 @@ ADD webpack.config.js app.json package.json package-lock.json tsconfig.json webp
 WORKDIR /app
 RUN npm install
 
+# Build API schema and types
+RUN npm run build-schema && \
```

Comment:
This should not be part of Dockerfile. These files are tracked by git. `build-schema` also depends on Django being installed in the system that runs the command.